src/components/atoms/Dropdown/Dropdown.tsx

```diff
@@ -38,6 +38,8 @@ const Dropdown = styled.select<IDropdownProps>`
 `;
 
 // TODO: Styleguidist to be able to locate styled components. See #147.
-export const StyleguidistDropdown: FC<IDropdownProps> = props => <Dropdown {...props} />;
+export const StyleguidistDropdown = forwardRef<HTMLSelectElement, IDropdownProps>((props, ref) => (
```

Comment:
We'll need to wrap `<Dropdown />` ( _default export below_ ) instead of this one as is the one exposed by the library 👍🏻

Reply:
done